README.md

```diff
@@ -40,6 +40,8 @@ async def main():
 asyncio.run(main())
 ```
 
+While you can provide an `api_key` argument, we recommend using python-dotenv to add LMNT_API_KEY="My API Key" to your .env file so that your API key is not stored in source control.
```

Comment:
nit: back-ticks around `python-dotenv`, `LMNT_API_KEY="My Api Key"`, and `.env`

Reply:
Done.